source/js/main.js

```diff
@@ -96,3 +96,30 @@ function createExploreRecipeCards(storageCategoryData) {
     })
   })   
 }
+
+
+/**
+ * active when search button click
+ * store Recipe to local with query as category name
+ * Does NOT apply any filter & sort, only search by name
+ * 1.add even handler to search button to fetch recipes for query 
+ * 2.add fetched recipes to local Storage
+ * 3.
```

Comment:
Please update the comments to be complete. The trailing '3.' concerns me, as that signals to me that you've missed something. 

Further, put a TODO for anything that needs to be done in the future.